pom.xml

```diff
@@ -85,7 +85,7 @@
         <mysql-connector-java.version>8.0.28</mysql-connector-java.version>
         <ojdbc8.version>12.2.0.1</ojdbc8.version>
         <poi.version>5.2.3</poi.version>
-        <postgresql.version>42.2.14</postgresql.version>
+        <postgresql.version>42.5.1</postgresql.version>
         <powermock.version>2.0.9</powermock.version>
```

Comment:
42.4.1? 42.5.1?

Reply:
I'm very sorry, my description is wrong, I have corrected it